src/json.jl

```diff
@@ -1,17 +1,14 @@
 # -------------------------------- #
 # Custom JSON output for our types #
 # -------------------------------- #
-JSON._print(io::IO, state::JSON.State, a::HasFields) =
-    JSON._print(io, state, a.fields)
+JSON.lower(a::HasFields) = a.fields
 
-JSON._print(io::IO, state::JSON.State, d::Base.Dates.Date) =
-    JSON._print(io, state, "$(Base.Dates.year(d))-$(Base.Dates.month(d))-$(Base.Dates.day(d))")
+JSON.lower(d::Base.Dates.Date) =
```

Comment:
Is there a reason you have this definition? I believe it is redundant with the one defined in JSON.jl.